src/pages/docs/webhooks.md

```diff
@@ -2,5 +2,13 @@
 title: Webhooks
 description: Getting important updates from Cirro.
 ---
+Webhooks are means of communication between apps; they provide a way in which apps convey information between each other. Each space can have one or multiple webhooks / configurations, each configuration containing the space's webhook url and a secret. 
```

Comment:
Cirro uses webhooks to communicate with its space. Currently cirro can inform its spaces about 2 events: user.updated and invitation.created.
user.updated event happens when user informations such as first name, last name or skills for an epamer are changed.